crates/typst-library/src/text/deco.rs

```diff
@@ -225,24 +225,79 @@ pub struct StrikeElem {
 impl Show for StrikeElem {
     #[tracing::instrument(name = "StrikeElem::show", skip_all)]
     fn show(&self, _: &mut Vt, styles: StyleChain) -> SourceResult<Content> {
+        let stroke = self.stroke(styles).unwrap_or_default();
         Ok(self.body().styled(TextElem::set_deco(Decoration {
-            line: DecoLine::Strikethrough,
-            stroke: self.stroke(styles).unwrap_or_default(),
+            // note that we do not support evade option for Strikethrough.
+            line: DecoLine::Strikethrough(stroke),
             offset: self.offset(styles),
             extent: self.extent(styles),
-            evade: false,
         })))
     }
 }
 
-/// Defines a line that is positioned over, under or on top of text.
+/// Highlight text by setting background color.
+///
+/// ## Example { #example }
+/// ```example
+/// This is #highlight[important].
+/// ```
+///
+/// Display: Highlight
+/// Category: text
+#[element(Show)]
+pub struct HighlightElem {
+    /// The color to fill in the background of the text.
+    /// (Default: Yellow)
+    ///
+    /// ```example
+    /// This is #highlight(fill: blue)[with blue]. \
+    /// ```
+    pub fill: Option<Paint>,
```

Comment:
```suggestion
    /// ```
    #[default(Color::YELLOW.into())]
    pub fill: Paint,
```